dockerfiles/bash/README.md

```diff
@@ -0,0 +1,59 @@
+# Experimental Bash language server 
+
+![](https://cl.ly/271p292i342p/Screen%20Recording%202018-05-07%20at%2009.19%20AM.gif)
```

Comment:
why no alt text?

Reply:
I wanted the line below it to be visible without needing to hover over the GIF, but feel free to move it